cmd/arcaflow/main.go

```diff
@@ -151,17 +152,20 @@ Options:
 		flag.Usage()
 		os.Exit(ExitCodeInvalidData)
 	}
-	cfg.Log.Stdout = os.Stderr
 
+	// now we are ready to instantiate our main logger
+	cfg.Log.Stdout = os.Stderr
 	logger := log.New(cfg.Log).WithLabel("source", "main")
 
-	var requiredFilesAbsSlice = make([]string, len(requiredFiles))
+	var requiredFilesAbsSlice = make([]string, len(requiredFilesAbsPaths))
+	var j int
 	for _, f := range requiredFilesAbsPaths {
-		requiredFilesAbsSlice = append(requiredFilesAbsSlice, f)
+		requiredFilesAbsSlice[j] = f
+		j++
 	}
```

Comment:
Why did you invent `j` here?  Doesn't it have exactly the same values as the first item returned by `range`?  I.e., wouldn't this be better?
```go
	for j, f := range requiredFilesAbsPaths {
		requiredFilesAbsSlice[j] = f
	}
```

Reply:
In your example, `j` is the key, and `f` is the value for the items in `requiredFilesAbsPaths`, whereas `requiredFilesAbsSlice` is an array that uses an integer index. Transforming a map's values into an array or list is still an [experimental feature](https://pkg.go.dev/golang.org/x/exp/maps#Values).